lib/private/twilio/send-message.js

```diff
@@ -0,0 +1,105 @@
+module.exports = {
+
+
+  friendlyName: 'Send message',
+
+
+  description: 'Send a message using the Twilio SMS API',
+
+
+  moreInfoUrl: 'https://www.twilio.com/docs/sms',
+
+
+  inputs: {
+
+    accountSid: {
+      example: 'DZe5eafd3c69b1e74example5852c04a9102',
+      description: 'The "account sid" associated with your Twilio account.',
+      whereToGet: {
+        url: 'https://www.twilio.com/console',
+      },
+      required: true
+    },
+
+    authToken: {
```

Comment:
can we change the input name to `secret` for consistency?

https://github.com/sailshq/sails-hook-organics/blob/e4223267dadf90c9f013c61b8de40481c91d25a7/lib/private/sendgrid/send-html-email.js#L48-L60

Reply:
done